metron-deployment/development/centos6/README.md

```diff
@@ -78,28 +80,103 @@ Any platform that supports these tools is suitable, but the following instructio
     vagrant provision
     ```
 
-### Explore Metron
+## Explore Metron
 
 Navigate to the following resources to explore your newly minted Apache Metron environment.
 
 * [Metron Alerts](http://node1:4201) credentials: user/password
 * [Ambari](http://node1:8080) credentials: admin/admin
 
 Connecting to the host through SSH is as simple as running the following command.
-```
-vagrant ssh
-```
+  ```
+  vagrant ssh
+  ```
+
+## Advanced Deployments
+
+In addition to running the entire provisioning play book, you can also use Ansible tags to limit the scope of what is deployed or deploy specific components.
+
+### Avoid an Extra Build
+
+If you have already built Metron, you can skip the build step when deploying the development environment.
+
+  1. Deploy the development environment without re-building Metron.
+      ```
+      vagrant --ansible-skip-tags="build,sensors,pcap" up
+      ```
+
+### Deploy Packet Capture
+
+If you want to deploy the components required to generate and capture network packets.
```

Comment:
Specific instructions for deploying the components required to test packet capture in the development environment.